html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-001.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>HTML: window.open `features`: tokenization -- `noopener`</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#apis-for-creating-and-navigating-browsing-contexts-by-name">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+var windowURL = 'resources/close-self.html';
+
+test (t => {
+  var featureVariants = [
+    'NOOPENER',
+    'noOpenER'
+  ];
+  featureVariants.forEach(feature => {
+    var win = window.open(windowURL, '', feature);
+    assert_equals(win, null, `"${feature}" should be tokenized -> name "noopener"`);
+  });
+}, 'feature name should be converted to ASCII lowercase');
+
+test (t => {
+  var featureVariants = [
+    '  noopener',
+    'noopener = ',
+    ' no opener',
```

Comment:
This test is wrong. Per the proposed spec as it stands right now this is equivalent to `no=opener`, but that's wrong also (https://github.com/whatwg/html/pull/2476/files#r109389922). I think this should result in name `no opener` and empty value (so unknown feature name). Let me try to fix the spec...